xbmc/addons/binary/interfaces/api1/PVR/AddonCallbacksPVR.cpp

```diff
@@ -437,14 +437,11 @@ void CAddonCallbacksPVR::PVREpgEventStateChange(void* addonData, EPG_TAG* tag, u
     return;
   }
 
-  CLog::Log(LOGDEBUG, "PVR - %s - state for epg event '%d' on channel '%d' on client '%s' changed to '%d'.",
-            __FUNCTION__, tag->iUniqueBroadcastId, iUniqueChannelId, client->Name().c_str(), newState);
-
   static CCriticalSection queueMutex;
   static std::vector<EpgEventStateChange> queuedChanges;
 
-  // during Kodi startup, addons may push updates very early, even before EPGs are ready to use.
-  if (g_PVRManager.EpgsCreated())
+  // during Kodi startup, addons may push updates very early, even before PVR manager and EPGs are ready to use.
+  if (g_PVRManager.IsStarted())
```

Comment:
not sure we should check both as the EPG gets loaded async and the PVR could be ready before the EPG was created.